static/operator.yaml

```diff
@@ -252,3 +252,11 @@ spec:
               value: "istio-operator"
 ...
 ---
+apiVersion: install.istio.io/v1alpha2
```

Comment:
This should still be removed as it leads to races. The docs should split up operator.yaml and the profile CRs.

Reply:
Some attention is needed here.  The first step is to revert the problems introduced by manual modification of these files.  There is automated tooling to do this work.

Reply:
Ok, I'll put this in and redo through the script.